lib/ddtrace/tracer.rb

```diff
@@ -440,6 +440,22 @@ def deactivate_priority_sampling!(base_sampler = nil)
       @sampler = base_sampler || Datadog::AllSampler.new if @sampler.is_a?(PrioritySampler)
     end
 
+    def to_h
+      {
+        :__class => self.class.name,
```

Comment:
Minor: 
* The default class `Module#to_s` is [mostly equivalent](https://rubyapi.org/3.0/o/module#method-i-to_s) to calling `.name` :)
* I'm curious about the `:__class`, since `{class: "something"}` seems to work...?
* Why `=>`, given they are all symbols? (same for others)

Reply:
1. I don't think this is true of some classes/modules, however, I do remember now its possible to have anonymous classes without a `name` so probably should adopt `to_s` for broader compatibility.
2. My idea here was to distinguish the different attributes by prefix: nothing for public methods, `@` for instance variables, and `__` in this case for metadata. The choice of `__` is arbitrary and not set in stone, but I prefer to differentiate between metadata and attributes by some convention.
3. You can't define a symbol key with `@worker: @worker`; it must be symbolized and paired with hash rockets.

Reply:
> I don't think this is true of some classes/modules, however, I do remember now its possible to have anonymous classes without a name so probably should adopt to_s for broader compatibility.

Oh, right! Good point `Class.new.name #=> nil` :)

> My idea here was to distinguish the different attributes by prefix: nothing for public methods, @ for instance variables, and __ in this case for metadata. The choice of __ is arbitrary and not set in stone, but I prefer to differentiate between metadata and attributes by some convention.

Ahaha I think the convention was too subtle, I didn't pick up on it. Perhaps worth documenting somewhere, if we do use it :)

> You can't define a symbol key with @worker: @worker; it must be symbolized and paired with hash rockets.

`{"@worker": @worker}` ;) ;) ;)

Reply:
Good catch... forgot you could quote like this (thought you had to do it with hash rockets.) Will make this adjustment.